packages/components/src/test/instance-filter-builder/PresentationFilterBuilder.test.tsx

```diff
@@ -422,11 +425,11 @@ describe("PresentationInstanceFilter", () => {
     expect(queryByDisplayValue(propertiesField.label)).to.not.be.null;
 
     // expand class selector
-    const expander = container.querySelector(".iui-actionable");
-    await user.click(expander!);
+    const expander = getByPlaceholderText("instance-filter-builder.selected-classes");
+    await user.click(expander);
 
     // deselect class item from dropdown
-    const classItem = document.querySelector(`li[label="${classInfo2.label}"]`);
+    const classItem = document.querySelector(`div[label="${classInfo2.label}"]`);
```

Comment:
Have you tried using `getByText` or similar alternative?

Reply:
Yes, that didn't work, explained in more detail privately.